github-stats.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+
+import json, requests, sys
+from datetime import datetime, timedelta
+
+# Fill in GitHub Token
+GITHUB_API_TOKEN = ''
+
+# Search for cards that are done and have been modified in the past 30 days
+GITHUB_ORG = 'redhat-cop'
+USER_AGENT= 'redhat-cop-stats'
+SEARCH_DAYS=30
```

Comment:
could we change (or add a new parameter) this to be since a certain date? That will be easier to calculate points within a given FY.

Reply:
We can change it to default to Feb 1 and give the user the ability to pass in a parameter. thoughts?